randovania/data/json_data/prime1.txt

```diff
@@ -999,141 +1644,498 @@ Burning Trail
 Triclops Pit
 Asset id: 3134844351
 > Dock to Monitor Tunnel; Heals? False
   * Normal Door to Monitor Tunnel/Dock to Triclops Pit
+  > Dock to Storage Cavern
+      All of the following:
+          Morph Ball
+          Any of the following:
+              Heated Rooms Damage ≥ 275
+              Boost Ball and Heated Rooms Damage ≥ 222
+  > Dock to Pit Tunnel
+      Any of the following:
+          Heated Rooms Damage ≥ 163
+          Space Jump Boots and Heated Rooms Damage ≥ 143
+  > Pickup (Missile Expansion)
+      All of the following:
+          Space Jump Boots and Heated Rooms Damage ≥ 201
+          Missile or Shoot Super Missile
+          X-Ray Visor or Invisible Objects (Intermediate)
 
 > Dock to Storage Cavern; Heals? False
   * Normal Door to Storage Cavern/Dock to Triclops Pit
+  > Dock to Monitor Tunnel
+      All of the following:
+          Morph Ball
+          Any of the following:
+              Heated Rooms Damage ≥ 299
+              Boost Ball and Heated Rooms Damage ≥ 286
+              Space Jump Boots and Heated Rooms Damage ≥ 263
+              Space Jump Boots and Boost Ball and Heated Rooms Damage ≥ 257
+  > Dock to Pit Tunnel
+      All of the following:
+          Morph Ball
+          Any of the following:
+              Heated Rooms Damage ≥ 267
+              Boost Ball and Heated Rooms Damage ≥ 239
+              Space Jump Boots and Heated Rooms Damage ≥ 242
+              Space Jump Boots and Boost Ball and Heated Rooms Damage ≥ 234
+
+> Dock to Pit Tunnel; Heals? False
+  * Normal Door to Pit Tunnel/Dock to Triclops Pit
+  > Dock to Monitor Tunnel
+      Any of the following:
+          Heated Rooms Damage ≥ 170
+          All of the following:
+              Morph Ball
+              Any of the following:
+                  Heated Rooms Damage ≥ 155
+                  Boost Ball and Heated Rooms Damage ≥ 152
+  > Dock to Storage Cavern
+      All of the following:
+          Morph Ball
+          Any of the following:
+              Heated Rooms Damage ≥ 252
+              Boost Ball and Heated Rooms Damage ≥ 211
+  > Pickup (Missile Expansion)
+      All of the following:
+          Space Jump Boots and Heated Rooms Damage ≥ 157
+          X-Ray Visor or Invisible Objects (Intermediate)
+          Missile or Shoot Super Missile
 
-> Dock to Pit Tunnel; Heals? False
-  * Normal Door to Pit Tunnel/Dock to Triclops Pit
+> Pickup (Missile Expansion); Heals? False
+  * Pickup 91; Major Location? False
+  > Dock to Monitor Tunnel
+      Any of the following:
+          Heated Rooms Damage ≥ 176
+          Space Jump Boots and Heated Rooms Damage ≥ 114
+  > Dock to Storage Cavern
+      All of the following:
+          Morph Ball
+          Any of the following:
+              Heated Rooms Damage ≥ 216
+              Boost Ball and Heated Rooms Damage ≥ 1188
+              No Space Jump Boots and Heated Rooms Damage ≥ 207
+              Space Jump Boots and Boost Ball and Heated Rooms Damage ≥ 177
```

Comment:
This node seems wrong. 
- What is the difference between the 1st path and 3rd path? Only 1st is needed.
- Is the 2nd path correct? That damage seems way too high.

Reply:
-3rd path is a mistake; I probably accidentally scrolled down on the comparison operator for space jump without realizing it.
-2nd path was indeed a typo, real value is 188. Thankfully I have these values documented elsewhere.